server/main.lua

```diff
@@ -6,73 +6,56 @@ local function showWarning(msg)
 	print(('^3%s: %s^0'):format(Lang:t("general.warning"), msg))
 end
 
+local function removeItem(Player, item)
+	if Config.Consumables[item.name] then
+		Player.Functions.RemoveItem(item.name, item.amount >= Config.Consumables[item.name] and Config.Consumables[item.name] or 1)
+	end
+end
+
+local function checkAndRemoveItem(Player, item, checkRemove)
+	if item then
```

Comment:
```suggestion
	if not item then return false end
```
You can change this if statement to this instead to remove a little unnecessary indentation and remove the `return false` at the end because it's not needed if you add this